.travis.yml

```diff
@@ -1,8 +1,11 @@
 language: go
 sudo: false
+arch:
+- amd64
+- ppc64le
 go:
-- 1.9.x
-- 1.10.x
+- 1.13.x
```

Comment:
Based on current Go releases, should we have 1.14.x and 1.15.x instead?

Reply:
Thanks! Update 1.14.x & 1.15.x